PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -299,11 +300,14 @@ open class PanModalPresentationController: UIPresentationController {
         longFormYPosition = layoutPresentable.longFormYPos
         
         if let scrollView = presentable?.panScrollable {
-            scrollView.contentInset.bottom = scrollView.contentInset.bottom - keyboardHeight
-            scrollView.scrollIndicatorInsets.bottom = scrollView.scrollIndicatorInsets.bottom + keyboardHeight
+            scrollView.contentInset = UIEdgeInsets.zero
```

Comment:
By resetting these to zero, you aren't accounting for genuine reasons someone may have custom content insets. Appreciate this is only for our specific use-cases at the moment, but if the goal is to make this generic, this wouldn't work in some situations. Up to you guys though.

Reply:
I kind of hear Mark's point, but as its on keyboard down, the content inset should perhaps be 0 anyway ( on the bottom anyway)

can we therefore, change this to just `scrollView.contentInset.Bottom` = 0 ? rather than set all insets?

Reply:
Why you assuming that contentInset bottom would be 0? Should surely be contentInset.bottom - keyboardHeight ?

Reply:
Check edits on 302, this is what it was, @haiderashfaq can you confirm why it was changed?

Reply:
Before i was taking into consideration a pre existing content inset, adding the keyboard height to this and then subtracting once keyboard hides. But this wasnt working, it gave me massive gaps under my content and funky behaviour on different devices. @TheiOSDude

Reply:
Ima try some stuff, give me a mo

Reply:
@TheiOSDude resolved in latest push, we now remember content inset. Tested on all use cases now, Im happy with it. LGTM